ui/raidboss/common_replacement.js

```diff
@@ -91,6 +91,20 @@ let commonReplacement = {
     cn: '--西北--',
     ko: '--북서--',
   },
+  '--orbs--': {
+    de: '--Kugeln--',
+    fr: '--orbes--',
+    ja: '--オーブ--',
+    cn: '--球--',
+    ko: '--오브--',
+  },
+  '--regulator check--': {
+    de: '--Dampfregler check--',
+    fr: 'vérification du régulateur',
+    ja: 'レギュレーターチェック',
+    cn: '--调节器检查--',
+    ko: '레귤레이터 점검',
+  },
```

Comment:
Sorry for the inconvenience, it's me again. Can you tell me, if this is correct (ja, cn, ko)? Thank you.
@Jaehyuk-Lee @r4ha

Reply:
Yes it is!

Reply:
Thank you.

Reply:
--orbs--
`ko: '--구슬--',`

--regulator check--
`ko: '--레귤레이터 확인--',`
I don't know what regulator is, but latter word is normally translated to "확인", "점검" is never been used in timelines.
I played since v3.26, and I didn't play savage before Alex:Creator so... it'd better other person who knows Korean check this, not me.